spec/features/teachers/user_can_login_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+
+describe "As a teacher" do
+  before :each do
+    @teacher = create(:teacher)
+    visit welcome_path
+  end
+
+  it "I can log in successfully" do
+    create(:course,2)
+    create(:student,4)
+    @teacher.courses << Course.first
+    Student.first.courses << Course.first
+    Student.second.courses << Course.first
+    Student.third.course << Course.second
+    Student.fourth.course << Course.second
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@teacher)
```

Comment:
If this test is for logging in, shouldn't `current_user` start as `nil`?

Reply:
I think the reason I put this was so the user existed, but I've realized that at this point, the database doesn't have any tables. This test is more of a rough draft because the db wasn't set up yet.